exhibitor_tls_artifacts/gen_artifacts.py

```diff
@@ -7,38 +7,42 @@
 
 
 @click.command()
-@click.argument('sans', nargs=-1)
+@click.option('--nodes', help='Comma separated list of master node ip addresses.')
```

Comment:
What will happen if I don't provide this flag? Should we turn it into a required argument?

Reply:
Good catch! It will break. I'll update the option to be required. Though that makes it less of an `option`.. Perhaps, should revert this back to being a positional argument?